docs/Installation Guide.md

```diff
@@ -6,8 +6,8 @@ This tutorial assumes that the user is installing and running the project under
 ## Table of Contents
 1. [Install git](https://github.com/systers/vms/blob/master/docs/Installation%20Guide.md#install-git)
 2. [Clone Project](https://github.com/systers/vms/blob/master/docs/Installation%20Guide.md#clone-project)
-2.5 [Install Python] (https://www.python.org/downloads/)
-2.6 [Install and make sure pip is working] (https://pip.pypa.io/en/latest/installing/)
+2.5 [Install Python] (https://github.com/systers/vms/blob/master/docs/Installation%20Guide.md#install-python)
+2.6 [Install and make sure pip is working] (https://github.com/systers/vms/blob/master/docs/Installation%20Guide.md#install-and-make-sure-pip-is-working)
```

Comment:
Mam, I did this in order to provide a direct link to the users to install python and check whether pip is working fine. I felt it will be helpful to the user.